### ISSUE TITLE
"View" button in the new posts ribbon not visible under gruvbox light theme

### ISSUE THREAD

**Issue by gatosardina** [Floens/Clover#332]
When there are new posts in a thread and the ribbon at the bottom appears, the view button is barely visible (font colour is almost the same as the ribbon color)
It happens only with gruvbox light theme, not gruvbox dark

**Comment by Floens** [comment]
@ente3000 can you look at this?

**Comment by ente3000** [comment]
@Floens Sure. I'm currently on the go and won't come home before 11 PM in my timezone (CEST), but I'll take a look at it tomorrow.

**Comment by ente3000** [comment]
Alright, this issue is resolvable by simply changing the color value at [line 204](https://github.com/ente3000/Clover/edit/dev/Clover/app/src/main/res/values/styles.xml#L204) to `#ebdbb2`.

However I'd rather like to remove the Gruvbox Light theme as it just didn't work with Clovers theme engine the way I hoped it would (see below screenshots).

When using the dark variant, dialogs and overflow menus both use correct background and text colors:

![screenshot_20170716-012640](https://user-images.githubusercontent.com/8181990/28246165-3b283730-6a14-11e7-8991-0c96cc532096.png)

However, when using the light one, they both use white as background color, which doesn't match with the theme at all.

![screenshot_20170716-012717](https://user-images.githubusercontent.com/8181990/28246164-3b24c2d0-6a14-11e7-8b86-af6c33d019c4.png)

This isn't a theme issue but rather an Android/Clover one as the background color of mentioned elements can't be themed but rather depends on whether `ThemeHelper.java` creates a new `Theme` or `DarkTheme`.

Long story short, I see two ways how to handle this:
1. Someone who knows more about Android programming than me enables the dialog and overflow menu background color to be variable, or my preferred one:
2. I remove the Gruvbox Light theme and rename the dark one to Gruvbox. I created a pull request doing exactly that [here](https://github.com/Floens/Clover/pull/335).

**Comment by gatosardina** [comment]
That's something I noticed too and was about to comment here
Too bad I guess, I enjoyed the gruvbox light theme

**Comment by Floens** [comment]
Hm, ok. But it really should be possible to style the menu and popup.
I'll merge the PR and maybe later we can add grvbox light back if it can look good.

**Comment by gatosardina** [comment]
well since this is moving nowhere in the near future i guess ill close it and open an issue to dd it back in some time
